index.css

```diff
@@ -237,6 +221,37 @@ h4 {
         order: 2;
     }
 
+    .doc-container {
+        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        margin: 0rem 4rem;
+    }
+
+    .side-nav-header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+    }
+
+    .side-nav {
+        height: 100%;
+    }
+       
+    .get-started-nav-items {
+        display: none;
+    }
+
+    .components-nav-items {
+        display: none;
+    }
```

Comment:
since the display: none style is common maybe u can use comma separator and write it like 
.get-started-nav-items , .componeents-nav-items {
    display: none;
}

Reply:
thank you @poornima-alugubelly for all your valuable inputs !!!